### ISSUE TITLE
Just a note that everything seems to be working nicely but had a question....

### ISSUE THREAD

**Issue by batjac7** [RandomEngy/VidCoder#221]
To get the most .... best mp4 video from whatever mkv video is running I have been manually selecting the same file size and encoding with that target size. I don't know a better way to do this. Is there a better way to try to get the best quality from the source? Some of the source video's are good and others, ... I get what I get. Is there some way to automate the process?   

Thanks

**Comment by RandomEngy** [comment]
Yes, you should use Quality targeting (which is used in all built-in presets). That will use just enough data to make the picture a certain quality.
